ffi/rust/examples/create_twin_did.rs

```diff
@@ -0,0 +1,36 @@
+use iotics_identity;
+
+/// Create a Twin DID
+/// Prerequisites: Create an User, an Agent and a delegation from the User to the Agent
+/// See the `create_user_and_agent` example
+fn main() {
+    // set up all the variables needed
+    let resolver_address = "https://did.stg.iotics.com";
+
+    let user_did = "did:iotics:iotNgaEk7erdRNZfuPczxFXs8M7LuYE8JVDk";
+
+    let agent_did = "did:iotics:iotKEeCnbYs9RM9DoPM9mdCK1At3qm7xvmwP";
```

Comment:
Did you leave these values of `user_did` and `agent_did` on purpose ?

Reply:
Yes. They are used for creating the auth token.